test/moonwall.config.json

```diff
@@ -19,7 +19,9 @@
                         "name": "tanssi",
                         "binPath": "../target/release/tanssi-node",
                         "options": [
-                            "--dev",
+                            "--alice",
+                            "--tmp",
+                            "--chain=tmp/dancebox_dev-raw.json",
```

Comment:
This can introduce race conditions to tests because `--dev` was also enabling some additional options.

The most important one is `--reserved-only` which prevents two nodes spawned at once from connecting to one another, as that will result in them running in a single network instead of each node being a separate network.

So I believe the args should be this:

```
                            "--chain=tmp/dancebox_dev-raw.json",
                            "--sealing=manual",
                            "--alice",
                            "--force-authoring",
                            "--rpc-cors=all",
                            "--tmp",
                            "--no-hardware-benchmarks",
                            "--no-prometheus",
                            "--no-telemetry",
                            "--reserved-only",
                            "--wasmtime-precompiled=wasm"
```